task/dev13/src/main/java/Pages/ChromeFailurePage.java

```diff
@@ -0,0 +1,29 @@
+package Pages;
+
+import Pages.Intarfaces.FailurePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Contains fail's page elements for Chrome browser.
+ */
+public class ChromeFailurePage implements FailurePage {
```

Comment:
ChromeFailurePage and FirefoxFailurePage differes only in one field, no need to create separate class